Client/src/Validation/error.js

```diff
@@ -0,0 +1,25 @@
+const buildErrors = (prev, id, error) => {
+	const updatedErrors = { ...prev };
+	if (error) {
+		updatedErrors[id] = error.details[0].message;
```

Comment:
What about providing a default error if this nested value isn't available for whaterver reason?

```
updatedErrors[id] = error?.details[0]?.message ?? "Validation error";
```

Something like that?